DOC-742: Document changes to lists plugin with multiple text blocks selected (#1823)
tinymce/tinymce-docs: release-notes/release-notes57.md

The following enhancements were made for the {{site.productname}} 5.7 release.

### Improved behavior when using `lists` plugin on mixed content

> The `lists` plugin has been updated to improve the handling of content containing:

Reviewer: Should this say something along the lines of "when converting content to lists"? As atm it's not very clear what "handling" is referring to here.

> Improved behavior when using the `lists` plugin to add lists to selections containing:

Reviewer 2: ```suggestion
The `lists` plugin has been updated to improve the handling of content containing:
```

Reviewer 2: Trying to make it more of a complete sentence? It reads really weird at the moment. feels too much like a changelog entry.

> - Multiple text blocks.

Reviewer: text blocks? That sounds confusing to me, as text can't be block elements.

> - Mixes of existing lists and text blocks.

Reviewer 3: ```suggestion
- Combinations of existing lists and text blocks.
```
`Mixes` just reads weird to me. Not entirely sure this is better though... @tylerkelly13

{{site.productname}} 5.7 introduces the following minor enhancements:

> - The `lists` plugin will now apply list styles to all text blocks within a selection.

Reviewer 3: Might it be worth mentioning that it applies to mixed paragraph/list content? Really not sure
